src/test/scala/BonusGeneratorSpec.scala

```diff
@@ -2,7 +2,7 @@ import org.scalatest.{MustMatchers, WordSpec}
 
 class BonusGeneratorSpec extends WordSpec with MustMatchers {
 
-  "BonusGenerator" must {
+  "BonusGenerator" ignore {
```

Comment:
@dougmac1 Nice work, however just to make you aware, if I checked out the project now and ran `sbt test` your ignored specs would not run. I'd recommend not committing an ignored spec.